### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by sangeethaE**

### CODE REVIEW

File: ansible-playbooks/psql.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+       db: sonar
+       name: sonar
+       password: "{{ sonarpassword }}"
+       priv: ALL
+       state: present
+	  ```
+	  ### Provide the password in Ansible Valut file and import the valut file in ansible playbook using vars_files module
+	```
+    vars_files:
+        - vault-pass.yml
```
Comment (RaveendraSeetharam) [inline ansible-playbooks/psql.md:78]: Could you check in this file?

**Review by RaveendraSeetharam — CHANGES_REQUESTED** [review]
Please submit related vault-pass.yml file?

File: ansible-playbooks/psql.md (hunk 1 above)
Comment (RaveendraSeetharam) [reply ansible-playbooks/psql.md:78]: Also, this file should not be checked into Documentation folder

**Comment by SangeethaEruparswapu** [comment]
Closing puu request to update the branching

**Comment by SangeethaEruparswapu** [comment]
close

**Pull request closed without merging**